Documentation/Signoffs/camcommdetaildesign.md

```diff
@@ -0,0 +1,50 @@
+# Camera Communications Detail Design
+
+
+## Big Picture
+
+
+
+1. The camera communications subsystem will interact directly with each camera as part of its build. This subsystem will allow each individual camera to connect to the small LAN(Local Area Network) that is provided by the server and its corresponding communications subsystem. This implies a need to provide each camera with the capability of connecting to WiFi. The chosen cameras have only ethernet (RJ45) inputs and thus require an adapter to connect them to a WiFi network.
+
+
+## Specifications

```

Comment:
No consideration of distance. Why are the capabilities of this module being given where the system constraints are meant to go?